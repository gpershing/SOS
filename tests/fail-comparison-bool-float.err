Fatal error: exception Failure("Cannot compare bool and float")
