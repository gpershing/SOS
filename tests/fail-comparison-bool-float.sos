b: bool = true > 0.1
