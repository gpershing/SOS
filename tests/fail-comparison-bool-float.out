Cannot compare float and bool
